lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -519,131 +644,135 @@ def _rows_for_users(self, context, users):
             return success_rows, error_rows
 
 
-class ProblemGradeReport(object):
+class ProblemGradeReport(GradeReportBase):
+    """
+    Class to encapsulate functionality related to generating Problem Grade Reports.
+    """
     @classmethod
     def generate(cls, _xmodule_instance_args, _entry_id, course_id, _task_input, action_name):
+        """
+        Public method to generate a grade report.
+        """
+        with modulestore().bulk_operations(course_id):
+            context = _ProblemGradeReportContext(_xmodule_instance_args, _entry_id, course_id, _task_input, action_name)
+            # pylint: disable=protected-access
+            return ProblemGradeReport()._generate(context)
+
+    def _generate(self, context):
         """
         Generate a CSV containing all students' problem grades within a given
         `course_id`.
         """
+        context.update_status('Starting grades')
```

Comment:
=> `Starting problem grades`

Reply:
Add some sort of differentiator, like `ProblemGradeReport Step 1/2/3`. It will make logs more distinctive